Library/Homebrew/brew.sh

```diff
@@ -222,7 +222,13 @@ EOS
 # exceeds 3 seconds.
 update-preinstall-timer() {
   sleep 3
-  echo 'Updating Homebrew...' >&2
+  # Outputting a command but don't want to run it, hence single quotes.
+  # shellcheck disable=SC2016
+  echo 'Running `brew update --preinstall`...' >&2
+  # shellcheck disable=SC2016
+  echo 'Adjust how often this is run with `HOMEBREW_AUTO_UPDATE_SECS` or disable with ' >&2
+  # shellcheck disable=SC2016
+  echo 'HOMEBREW_NO_AUTO_UPDATE (see `brew man`) ' >&2
```

Comment:
Agreed 👍🏻

Reply:
In messages I've kept backticks off of env vars (regex search: `^(?> +)[^#].* HOMEBREW(_[A-Z]+)+ `), since they imply a whole or partial command that might be typed.